lib/LaTeXML/resources/CSS/LaTeXML.css

```diff
@@ -59,7 +59,9 @@
 
 /* .ltx_chapter_title, etc should be in ltx-article.css etc.
  */
-.ltx_page_main { margin:0px; padding:1em 3em 1em 2em; }
+.ltx_page_main { margin:0px auto; max-width: 36em; max-width: 72ch;
+    padding:1em 3em 1em 2em;
+    padding:min(1em,1.5%) min(3em,4.5%) min(1em,1.5%), min(2em, 3%); }
```

Comment:
Also: Isn't the "," after the 3rd min clause a typo? (same in the previous two .css)

Reply:
Must have been a bad day when I submitted this... yes, the last comma is a typo. The line is supposed to replicate the previous one with the extra min(..., ...%).

And now that I see this, I should have documented the magic "1.5%" number. It's chosen so that the old 1em wins when the width is around 1100px.